cmd/k3d/create.go

```diff
@@ -54,10 +55,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var (
-	cancelContext context.CancelFunc
```

Comment:
This appears unused in the rest of the codebase. There was one below but was being shadowed.